### PR TITLE
Bug fix for GAI-HostInfo: AzSK description

### PR DESCRIPTION
**Pull request by ritika-msft**
## Description
</br>
 Brief description of the requested change.

## Checklist
- [ ] I have read the instructions mentioned in [Contribute to Code](/Contributing.md).
- [ ] I have read and understood the criteria described under [submitting changes](/Contributing.md#submitting-changes). 
- [ ] The title of the PR clearly describes the intent of the PR.
- [ ] This PR does not introduce any breaking changes to the code.

### CODE REVIEW

File: src/AzSK.AzureDevOps/AzSK.AzureDevOps.psd1 [hunk 1]
```diff
@@ -27,7 +27,7 @@
     Copyright              = '(c) 2017 Microsoft Corporation. All rights reserved.'
 
     # Description of the functionality provided by this module
-    Description            = 'DevSecOps Kit for AzureDevOps(AzSK) - Preview'
+    Description            = 'Secure DevOps Kit for Azure (AzSK) - Preview'
```
Comment (v-zhgup) [inline src/AzSK.AzureDevOps/AzSK.AzureDevOps.psd1]: Should it be Secure DevOps Kit for AzureDevOps

File: src/AzSK/AzSKStaging.psd1 [hunk 2]
```diff
@@ -12,7 +12,7 @@
     RootModule             = '.\AzSK.psm1'
 
     # Version number of this module.
-    ModuleVersion          = '4.0.0.0'
+    ModuleVersion          = '4.4.4'
```
Comment (VishalHaibatpure) [inline src/AzSK/AzSKStaging.psd1]: Do not change this module version. Base version 4.0.0.0 is used in release configuration to replace it with actual version

File: src/AzSK.AzureDevOps/AzSK.AzureDevOps.psd1 (hunk 1 above)
Comment (ritika-msft) [reply src/AzSK.AzureDevOps/AzSK.AzureDevOps.psd1]: Discussed this with Abhay, he suggested to keep it "Secure DevOps Kit for Azure".

File: src/AzSK/AzSKStaging.psd1 (hunk 2 above)
Comment (ritika-msft) [reply src/AzSK/AzSKStaging.psd1]: Changed the Module version back to '4.0.0.0' in the latest commit

**Pull request merged**